ante/fee-notes.md

```diff
@@ -0,0 +1,53 @@
+# Notes about Gas prices and gas used
+
+NOTES:
+
+- the gas used depends on the current state of the chain
+- the gas records below are based on the empty state
+- user is charged the gas-limit, even if tx consumed less gas
```

Comment:
Why do we charge the gas limit, not the gas consumed?

Reply:
thanks for the explanation, can we add that to the fee-notes.md 
or create an issue to track this and not forget

Reply:
OK. I will update fee-notes.md